ext/sqlite3/sqlite3.stub.php

```diff
@@ -60,13 +60,13 @@ public function query(string $query) {}
     public function querySingle(string $query, bool $entire_row = false) {}
 
     /** @return bool */
-    public function createFunction(string $name, $callback, int $argument_count = -1, int $flags = 0) {}
+    public function createFunction(string $name, callable $callback, int $argument_count = -1, int $flags = 0) {}
```

Comment:
I accidentally added the `callable` type in these methods. Although I'm wondering if the `zend_is_callable()` check used in these functions really work despite of https://bugs.php.net/bug.php?id=78770.